src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -203,6 +206,7 @@ public JabRefFrame(Stage mainStage) {
         });
     }
 
+    @SuppressWarnings("checkstyle:SingleSpaceSeparator")
```

Comment:
This is not the right way to handle checkstlye issues..

Reply:
I do not belive that I have added this.